### PR TITLE
Add sentry

### PR DESCRIPTION
**Pull request by KedziaPawel**

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ YellowBox.ignoreWarnings(['VirtualizedLists should never be nested inside', `\`-
 
 if (process.env.NODE_ENV !== 'development') {
   Sentry.init({
-    dsn: 'https://23377936131848ca8003448a893cb622@sentry.io/1295736',
+    dsn: 'https://dc67fd6d5c2949f2a6853e60eb69d899@o429172.ingest.sentry.io/5375289',
```
Comment (mateuszpmroz) [inline App.tsx:20]: maybe const or env?

Comment (KedziaPawel) [reply App.tsx:20]: env seems to be a good idea

File: App.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ YellowBox.ignoreWarnings(['VirtualizedLists should never be nested inside', `\`-
 
 if (process.env.NODE_ENV !== 'development') {
```
Comment (Kamill90) [inline App.tsx:18]: `!__DEV__` would be enough

File: App.tsx (hunk 1 above)
Comment (KedziaPawel) [reply App.tsx]: moved to config

File: App.tsx (hunk 2 above)
Comment (KedziaPawel) [reply App.tsx]: done

**Pull request merged**